modules/mill-codegen-plugin/test/src/smithy4s/codegen/mill/SmithyLSPConfigSpec.scala

```diff
@@ -0,0 +1,58 @@
+package smithy4s.codegen.mill
+
+import mill._
+import mill.scalalib._
+import mill.testkit.MillTestKit
+import sourcecode.FullName
+
+class SmithyLSPConfigSpec extends munit.FunSuite {
```

Comment:
thought: to be precise it's the smithy-cli config, the LSP happens to use it

Reply:
![](https://imgflip.com/i/80to1e)

Reply:
I missed this 😂 

it was a good comment. If you use smithy-cli (which has some nice things in it), you won't think to use `smithyLspConfig`